docs/sphinx/user_guide/feature/view.rst

```diff
@@ -92,21 +92,62 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+Reshape
+^^^^^^^
+The ``RAJA::Reshape`` method takes a pointer to data and extents in each dimension as arguments and returns a ``RAJA::View`` object.
+The reshape method is specialized on a `meta-layout` template parameter, which specifies the stride ordering for the dimensions.
+
+The first meta-layout is based on a right-most layout which follows standard C indexing (right most index has unit stride).
+For example:
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_right_start
+   :end-before: _Reshape_right_end
+   :language: C++
+In the example above a ``RAJA::View`` is returned with right most indexing having unit stride.
+
+The second supported meta-layout is the ``RAJA::layout_left`` which will follow a Fortran style indexing
+in which the left most index has unit stride and the right most has the longest stride. The configuration
+is demonstrated in the following example.
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_left_start
+   :end-before: _Reshape_left_end
+   :language: C++
+
+Finally, users may specify a custom index ordering by providing a ``std::index_sequence`` type as a
+template argument to the ``RAJA::Reshape`` method. The members of the ``std::index_sequence`` enumerate
+the striding order of the arguments of the ``RAJA::View`` parenthesis operator.
+
+For example to use the C index ordering (``RAJA::layout_right``) with a 3-dimensional ``RAJA::View`` use the following 
+sequence type ``std::index_sequence<0U,1U,2U>``.  In the following example, we consider a permutation

```

Comment:
I think this would be clearer if you added a sentence that states exactly what the indexing order is for this case.